tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -361,3 +361,21 @@ def wait_for_condition(client, resource, check_function, fail_handler=None,
         resource = client.reload(resource)
 
     return resource
+
+
+@pytest.fixture
+def wait_until_expected_state(client, resource, expected_state,
```

Comment:
I will remove wait_until_expected_state and use wait_for_condition function that son has added